frontend/src/components/OrganizationProfilePage/OrganizationProfilePage.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import Container from '@material-ui/core/Container';
+import { Typography, Box } from "@material-ui/core";
+
+import { OrganizationType } from '../../types/OrganizationType';
+
+const styles = {
+  container: {
+    display: "flex",
+    padding: "100px",
+  }
+}
+
+type Props = {
+  location: {
+    state: {
```

Comment:
why location.state?

Reply:
This is just how props get passed via the Redirect element - https://stackoverflow.com/questions/52064303/reactjs-pass-props-with-redirect-component